# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fedcm/lfedcm-identity.create-store-collect.tentative.sub.https.html.ini
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 testing/web-platform/meta/fedcm/lfedcm-identity.create-store-collect.tentative.sub.https.html.ini
--- a/testing/web-platform/meta/fedcm/lfedcm-identity.create-store-collect.tentative.sub.https.html.ini	Wed Jun 11 13:42:13 2025 +0000
+++ b/testing/web-platform/meta/fedcm/lfedcm-identity.create-store-collect.tentative.sub.https.html.ini	Wed Jun 11 18:04:27 2025 +0300
@@ -1,7 +1,6 @@
 [lfedcm-identity.create-store-collect.tentative.sub.https.html]
   expected:
     if (os == "mac") and not debug: [OK, TIMEOUT]
-    TIMEOUT
   [Collecting from no providers yields a promise with null resolution]
     expected: FAIL
 
@@ -11,45 +10,7 @@
   [Credentials can be collected silently, but preventSilentAccess is respected]
     expected:
       if (os == "mac") and not debug: [PASS, TIMEOUT, NOTRUN]
-      NOTRUN
 
   [Cross-origin identity credential collection is correctly filtered when the type does not match]
     expected:
       if (os == "mac") and not debug: [PASS, TIMEOUT]
-      NOTRUN
-
-  [Basic valid credential creation]
-    expected: FAIL
-
-  [Basic invalid credential creation]
-    expected: FAIL
-
-  [Basic identity credential store suceeds]
-    expected: FAIL
-
-  [Basic same-origin identity credential collect with effectiveQueryURL suceeds]
-    expected: FAIL
-
-  [Basic same-origin identity credential collect with origin inferred from loginURL succeeds]
-    expected: FAIL
-
-  [Basic same-origin identity credential collect with effectiveQueryURL and no ACAO works because CORS is not needed]
-    expected: FAIL
-
-  [Cross-origin identity credential collection is correctly filtered when there is no effective argument]
-    expected: TIMEOUT
-
-  [Cross-origin identity credential collection is correctly filtered when the effective origin is not the relying party]
-    expected: NOTRUN
-
-  [Cross-origin identity credential collection is correctly returned when the effective origin is the relying party]
-    expected: NOTRUN
-
-  [Cross-origin identity credential collection is correctly filtered when the endpoint doesn't have CORS]
-    expected: NOTRUN
-
-  [Cross-origin identity credential collection is correctly returned when the endpoint returns success]
-    expected: NOTRUN
-
-  [Cross-origin identity credential collection is correctly returned when the type matches]
-    expected: NOTRUN